src/gui/Src/Gui/MemoryMapView.cpp

```diff
@@ -444,6 +446,11 @@ void MemoryMapView::refreshMapSlot()
         setCellContent(i, ColAddress, ToPtrString((duint)mbi.BaseAddress));
         setCellUserdata(i, ColAddress, (duint)mbi.BaseAddress);
 
+        if((duint)mbi.BaseAddress == currSelectedAddressRow)
+        {
+            oldAddressFound = true;

```

Comment:
I don't think it's necessary to delay setting the selection and storing the state. You can directly call `setSingleSelection` here to simplify everything. Also the variable name is no longer accurate, it should be something like `prevSelectedBase`

Reply:
the `reloadData` function below caused some problems with `setSingleSelection` (it was jumping around like crazy)